### ISSUE TITLE
can find the results of combine_A_and_B.py

### ISSUE THREAD

**Issue by 1987521** [junyanz/pytorch-CycleGAN-and-pix2pix#1515]
At first, the combination of two kinds picture can be stored in the fold_AB. However, after several times trying, the code is working properly, the results didn't show in the fold_AB that I created

**Comment by maryam95hallal** [comment]
Hello The following worked for me:
The images in folder A and B must have the same name and size
set the flag --fold_AB to an existing folder

**Comment by 1987521** [comment]
> Hello The following worked for me: The images in folder A and B must have the same name and size set the flag --fold_AB to an existing folder

yes, i found the problems that the folder A and B didn't have the same

**Comment by maryam95hallal** [comment]
you can either resize the image b or add padding or empty pixels
